src/shared/codelens/codeLensUtils.ts

```diff
@@ -20,7 +20,7 @@ import { defaultMetricDatum } from '../telemetry/telemetryUtils'
 import { toArrayAsync } from '../utilities/collectionUtils'
 import { localize } from '../utilities/vsCodeUtils'
 
-export type Language = 'python' | 'javascript'
+export type Language = 'python' | 'javascript' | 'csharp'
```

Comment:
Nit: should we rename `typescriptCodeLensProvider` to `javascriptCodeLensProvider`?

Reply:
Out of scope for the feature branch merge